packages/aws-cdk-lib/aws-eks/README.md

```diff
@@ -713,10 +713,12 @@ By default, CDK will create a new python lambda function to apply your k8s manif
 
 ```ts
 const handlerRole = iam.Role.fromRoleArn(this, 'HandlerRole', 'arn:aws:iam::123456789012:role/lambda-role');
+const serviceToken = lambda.Function.fromFunctionName(this, 'ProviderOnEventFunc', 'ProviderframeworkonEvent-XXX').functionArn;
 const kubectlProvider = eks.KubectlProvider.fromKubectlProviderAttributes(this, 'KubectlProvider', {
   functionArn: 'arn:aws:lambda:us-east-2:123456789012:function:my-function:1',
   kubectlRoleArn: 'arn:aws:iam::123456789012:role/kubectl-role',
   handlerRole,
+  serviceToken,
```

Comment:
If users will have to change their code anyway, why can't we tell them to pass the service token (as created above) to `functionArn`?

Reply:
Ahhhh! Yes you are right! lol